### PR TITLE
0.4.1

### PR DESCRIPTION
**Pull request by sudoryan**
New release to include change that removes unused ELA versions

### CODE REVIEW

File: contracts/UNSRegistry.sol [hunk 1]
```diff
@@ -29,7 +29,7 @@ contract UNSRegistry is
     IUNSRegistry
 {
     string public constant NAME = 'UNS: Registry';
-    string public constant VERSION = '0.4.0';
+    string public constant VERSION = '0.4.1';
```
Comment (aquiladev) [inline contracts/UNSRegistry.sol:32]: no need to change the version

File: uns-config.json [hunk 2]
```diff
@@ -1,5 +1,5 @@
 {
-    "version": "0.4.0",
+    "version": "0.4.1",
```
Comment (aquiladev) [inline uns-config.json:2]: no need to change the version

**Review by aquiladev — CHANGES_REQUESTED** [review]
these changes are not required, if you do not plan to release new tag

**Comment by sudoryan** [comment]
I guess I don't understand how the tag system works on this repo. I'd like to be able to use the updated resolver-keys.json on the website. Are these changes already included in v0.4.0 or do I need to create a new tag and bump the version in the website?

**Review by aquiladev — APPROVED** [review]
ok, then these changes are needed.
there is the process:
1. merge changes with version update to the master
1. create and push new tag
  ```
  git tag v0.4.1
  git push origin --tags
  ```
3. update **uns** package with new tag on website
  ```
  yarn remove uns 
  yarn add https://github.com/unstoppabledomains/uns#v0.4.1
  ```

**Pull request merged**